packages/permissions/src/components/authorized/authorized.spec.tsx

```diff
@@ -283,8 +283,8 @@ describe('rendering', () => {
             });
             shallow(<Authorized {...props} />);
           });
-          it('should indicate as not authorized', () => {
-            expect(props.render).toHaveBeenCalledWith(false);
+          it('should indicate as authorized', () => {
+            expect(props.render).toHaveBeenCalledWith(true);
```

Comment:
Just double checking: is this the expected outcome now?

Reply:
yes